cloudweatherreport/cloud_weather_report.py

```diff
@@ -195,7 +195,7 @@ def main(args, test_plan):
             continue
         last_successful_status = status
         benchmark_results = []
-        if test_plan.get('benchmark'):
+        if status is not None and test_plan.get('benchmark'):
```

Comment:
Cool. You might want to leave a comment along the lines of "otherwise, skip benchmarks due to Exception in unit tests" on that line with the "is not None" check for status, to prevent the purpose of the check from fading into obscurity in a few months :-)